src/python/pants/pantsd/service/scheduler_service.py

```diff
@@ -162,7 +162,7 @@ def product_graph_len(self):
     """
     return self._scheduler.graph_len()
 
-  def prefork(self, options, options_bootstrapper):
+  def prepare_graph(self, options, options_bootstrapper):
     """Runs all pre-fork logic in the process context of the daemon.
```

Comment:
Clarifying this comment would be helpful, and might help to name these. I think that what the `loop` and `body` are now doing is:
1. For v1 (and v2): computing TargetRoots for a later v1 run
2. For v2: running an entire v2 run

Reply:
Updated.